devdoc/umock_lock_factory_windows.md

```diff
@@ -0,0 +1,97 @@
+# umock_lock_factory_windows
+
+## Overview
+
+`umock_lock_factory_windows` is a module that implements the lock factory for `umock_c` for Windows.
+
+## Exposed API
+
+```c
+    UMOCK_C_LOCK_HANDLE umock_lock_factory_create_lock(void* params);
+```
+
+## static lock functions
+
+```c
+static void umock_lock_windows_acquire_shared(UMOCK_C_LOCK_HANDLE lock);
+static void umock_lock_windows_release_shared(UMOCK_C_LOCK_HANDLE lock);
+static void umock_lock_windows_acquire_exclusive(UMOCK_C_LOCK_HANDLE lock);
+static void umock_lock_windows_release_exclusive(UMOCK_C_LOCK_HANDLE lock);
+static void umock_lock_windows_release_destroy(UMOCK_C_LOCK_HANDLE lock);
```

Comment:
>_release [](start = 30, length = 8)

probably just destroy without release

Reply:
Fixed.

---
In reply to: [530647559](https://github.com/Azure/umock-c/pull/177#discussion_r530647559) [](ancestors = 530647559)